Update About-Actions.rst (#3928)
ros2/ros2_documentation: source/Concepts/Basic/About-Actions.rst

-------------

The action server is the entity that will accept the remote procedure request and perform some procedure on it.
It is also responsible for sending out feedback as the action progresses and should react to cancellation/preemption requests.

> For instance, suppose an action is considered to calculate the Fibonacci sequence, and the message contains the following:

Reviewer: "considered" isn't really appropriate in this sentence.  How about:

```suggestion
For instance, consider an action to calculate the Fibonacci sequence with the following interface:
```
